### ISSUE TITLE
UI Crash

### ISSUE THREAD

**Issue by ghost** [TeamNewPipe/NewPipe#4057]
### Version

- 0.19.8

### Steps to reproduce the bug
1. Scroll down a channel really quickly!!

## Exception
* __User Action:__ ui error
* __Request:__ App crash, UI failure
* __Content Country:__ US
* __Content Language:__ en
* __App Language:__ en_US
* __Service:__ none
* __Version:__ 0.19.8
* __OS:__ Linux Android 9 - 28
<details><summary><b>Crash log </b></summary><p>

```
java.lang.IllegalStateException: ViewHolder views must not be attached when created. Ensure that you are not passing 'true' to the attachToRoot parameter of LayoutInflater.inflate(..., boolean attachToRoot)
	at androidx.recyclerview.widget.RecyclerView$Adapter.createViewHolder(RecyclerView.java:7080)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6235)
	at androidx.recyclerview.widget.GapWorker.prefetchPositionWithDeadline(GapWorker.java:288)
	at androidx.recyclerview.widget.GapWorker.flushTaskWithDeadline(GapWorker.java:345)
	at androidx.recyclerview.widget.GapWorker.flushTasksWithDeadline(GapWorker.java:361)
	at androidx.recyclerview.widget.GapWorker.prefetch(GapWorker.java:368)
	at androidx.recyclerview.widget.GapWorker.run(GapWorker.java:399)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

```
</details>
<hr>

**Comment by Stypox** [comment]
I don't think this can be fixed. NewPipe's code is already doing what the error suggests to do, i.e. `Ensure that you are not passing 'true' to the attachToRoot parameter of LayoutInflater.inflate(..., boolean attachToRoot)`. So the issue could be that you were scrolling so fast that the system could not keep up and ended up asking for a video info item view still not fully constructed, thus generating this crash. I can't reproduce unfortunately

**Comment by ingingin** [comment]
@Stypox Yeah, I can't reproduce this issue myself anymore. The problem occured when I scrolled down fast while the videos took time to load. Even more new videos loaded in without loading the thumbnail for the other videos and I could basically scroll very fast and so everything couldn't keep up. And eventually crashed, but maybe it was a network related crash. 

**TLDR**; Before, I could scroll faster than the videos loaded, thus making it crash I guess. Now I can't scroll that fast anymore...

**Comment by Stypox** [comment]
Closing as not reproducible
